server/api/assets.js

```diff
@@ -126,11 +130,13 @@ router.post(
           sequelize.asset.create(
             {
               fingerprint: hash256Bytes,
+              height,
               ipfs: bs58.decode(ipfsAdd[0].hash),
               ipld: bs58.decode(ipld.toBaseEncodedString()),
               license,
               tags: [{ name: 'hehe' }, { name: 'haha' }],
               wallet,
+              width,
             },
```

Comment:
Missing `description`?

Reply:
Are tags ready for assets upload? Since I see it saves `hehe` and `haha` instead of the real tags submitted by user

Reply:
Updated thanks, I have not tested it as there's no UI now.